Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -150,6 +150,17 @@ inline void ClearStrokeStyle() {
         return GStateStack().top();
     }
 
+    inline bool HasPath() {
+        return _impl.currentPath != nullptr;
+    }
+
+    inline CGMutablePathRef Path() {
+        if (!_impl.currentPath) {
+            _impl.currentPath.reset(CGPathCreateMutable());
+        }
```

Comment:
Correct; CGContext is not explicitly thread-safe.

---
In reply to: [85250507](https://github.com/Microsoft/WinObjC/pull/1240#discussion_r85250507) [](ancestors = 85250507)